code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -165,13 +165,13 @@
 	amount_per_transfer_from_this = 2
 	stream_amount = 5
 
-/obj/item/reagent_containers/spray/cleaner/suicide_act(mob/user)
+/obj/item/reagent_containers/spray/suicide_act(mob/user)

```

Comment:
No code changes needed but the code for this proc is in the wrong place in the file. Just put it above the cleaner spray bottle definition with the rest of the spray bottle procs